.github/workflows/swift-toolchain.yml

```diff
@@ -747,78 +738,38 @@ jobs:
           path: ${{ github.workspace }}/SourceCache/swift-driver
           show-progress: false
 
-      - uses: compnerd/gha-setup-vsdevenv@f1ba60d553a3216ce1b89abe0201213536bc7557 # main
-        with:
-          host_arch: ${{ inputs.build_arch }}
-          components: 'Microsoft.VisualStudio.Component.VC.Tools.x86.x64;Microsoft.VisualStudio.Component.VC.Tools.ARM64'
-          arch: ${{ matrix.arch }}
-
-      - uses: seanmiddleditch/gha-setup-ninja@96bed6edff20d1dd61ecff9b75cc519d516e6401 # master
-        if: inputs.build_os == 'Darwin'
-
-      - name: Install Swift Toolchain (Mac)
-        if: inputs.build_os == 'Darwin'
+      - name: Install Swift Toolchain
         uses: compnerd/gha-setup-swift@b6c5fc1ed5b5439ada8e7661985acb09ad8c3ba2 # main
         with:
           branch: swift-6.0.1-release
           tag: 6.0.1-RELEASE
 
       - name: Build early swift-driver
         run: |
-          swift build `
-                --configuration release `
-                --arch ${{ matrix.arch }} `
-                --package-path ${{ github.workspace }}/SourceCache/swift-driver `
-                --build-path ${{ github.workspace }}/BinaryCache/swift-driver `
-
-      - name: Create installable package
-        run: |
-          if ("${{ matrix.os }}" -eq "Darwin") {
-              $PlatformName = "macosx"
-              $ExeSuffix = ""
-              $Cpu = "${{ matrix.arch }}"
-              $SourceBinDir = Join-Path "${{ github.workspace }}" "BinaryCache" "swift-driver" "${Cpu}-apple-macosx" "release"
-          } else {
-              $PlatformName = "windows"
-              $ExeSuffix = ".exe"
-              if ("${{ matrix.arch }}" -eq "ARM64") {
-                $Cpu = "aarch64"
-              } else {
-                $Cpu = "x86_64"
-              }
-              $SourceBinDir = Join-Path "${{ github.workspace }}" "BinaryCache" "swift-driver" "${Cpu}-unknown-windows-msvc" "release"
-          }
-
-          # The Swift compiler expects a specific folder name for the early swift-driver build.
-          # TODO: Make this configurable upstream so we don't have to hardcode it here.
-          $InstallFolderName = "earlyswiftdriver-${PlatformName}-${Cpu}"
-          $InstallBinDir = Join-Path "${{ github.workspace }}" "BinaryCache" $InstallFolderName "release" "bin"
-
-          # Create the target folder.
-          New-Item -ItemType Directory -Path $InstallBinDir -Force
+          $env:SWIFTCI_USE_LOCAL_DEPS=1
 
-          # Copy binaries.
-          $SwiftBinaries = @("swift-driver", "swift-help")
-          foreach ($bin in $SwiftBinaries) {
-              $binName = "${bin}${ExeSuffix}"
-              $binPath = Join-Path $InstallBinDir $binName
-              Copy-Item -Path "${SourceBinDir}/${binName}" -Destination $binPath -Force
-          }
```

Comment:
We are going to need `gha-upload-tar-artifact` for the Mac build, which only takes a single path or file. I would suggest doing changes here similar to what is done in my [cross PR](https://github.com/compnerd/swift-build/pull/851/files) but with a saner directory name.

Reply:
Let's get this building on Windows first. Once we have that, we can address the use of gha-upload-tar-artifact.